packages/lib-classifier/src/plugins/drawingTools/models/marks/Circle/Circle.js

```diff
@@ -24,8 +24,8 @@ const CircleModel = types
 
     deleteButtonPosition(scale) {
       const theta = DELETE_BUTTON_ANGLE * (Math.PI / 180)
-      const dx = ((self.r + BUFFER) / scale) * Math.cos(theta)
-      const dy = ((self.r + BUFFER) / scale) * Math.sin(theta)
+      const dx = (self.r + BUFFER) * Math.cos(theta)
+      const dy = (self.r + BUFFER) * Math.sin(theta)
```

Comment:
These changes are technically unrelated, but fixes a small bug, similar to https://github.com/zooniverse/front-end-monorepo/pull/6065#discussion_r1684209230